infra/buckets/s3.tf

```diff
@@ -150,3 +150,12 @@ resource aws_s3_bucket dss_s3_checkout_bucket_unwritable {
 }
 POLICY
 }
+
+resource aws_s3_bucket dss_s3_events_bucket {
+  count = "${length(var.DSS_S3_BUCKET) > 0 ? 1 : 0}"
```

Comment:
does the var evaluated in the count need to be the event bucket?

Reply:
yes, fixed